gammapy/irf/io.py

```diff
@@ -124,3 +129,49 @@ def load_cta_irfs(filename):
     psf = EnergyDependentMultiGaussPSF.read(filename, hdu="POINT SPREAD FUNCTION")
 
     return dict(aeff=aeff, bkg=bkg, edisp=edisp, psf=psf)
+
+
+def load_irf_dict_from_file(filename):
```

Comment:
An issue with this function is what happens if several IRFs of the same type are stored in the same file, e.g. same run, different event type.
A minima, one should document the behavior and log some warning in case there is a doubt.

Reply:
I added a Warning message if two irf with the same HDU class type are detected.
It specifies the `EXTNAME` of the one that has been loaded in the dictionary.